### ISSUE TITLE
Machine readable output

### ISSUE THREAD

**Issue by DorianCzichotzki** [darrenburns/ward#130]
I was looking into creating a Visual Studio Code extension for ward. It's hard to parse the output that is printed to the terminal. A way to let ward output json (or any other structured format) would help to create integrations for all kinds of tools.

**Comment by darrenburns** [comment]
Oh, this would be very cool. Let me know if you get started on the extension, I'd love to see it and maybe help out if required!

I'm going to close this in favour of #123, since it covers the same functionality, but I'd be more than willing to accept a PR that added something like an `--output-json` or something similar (happy to discuss the API -- haven't put much thought into it).
